src/index.mjs

```diff
@@ -34,7 +34,7 @@ export default class ImgixClient {
     }
 
     if (this.settings.includeLibraryParam) {
-      this.settings.libraryParam = 'js-' + VERSION;
+      this.settings.libraryParam = 'js-core-' + VERSION;
```

Comment:
Is this necessary if the version number is staying on track with what we currently have?

Reply:
No it's not necessary, but I had wondered what you thought about it. I'm good with keeping just the `js-` portion.